codalab/rest/interpret.py

```diff
@@ -218,8 +248,13 @@ def fetch_interpreted_worksheet(uuid):
                 if not is_relevant_block:
                     interpreted_blocks['blocks'][i] = None
 
+    # items: actual blocks in the worksheets, each item may group several raw items
+    #        into one block i.e one table item groups multiple bundle raw items
     worksheet_info['items'] = resolve_interpreted_blocks(interpreted_blocks['blocks'])
+    # raw_to_block: an array that records how raw items are grouped into items.
+    # The value is of form [block_index, sub_block_index]
     worksheet_info['raw_to_block'] = interpreted_blocks['raw_to_block']
+    # block_to_raw: inverse mapping of raw_to_block
```

Comment:
You can remove these inline comments now that you've added this info to the docstring at the beginning.